src/js/components/Button/StyledButton.js

```diff
@@ -99,8 +99,8 @@ export const StyledButton = styled.button`
 
   ${genericStyles}
   ${props => props.plain && plainStyle}
-  ${props => !props.plain && basicStyle(props)}
   ${props => props.primary && primaryStyle(props)}
+  ${props => !props.plain && basicStyle(props)}
```

Comment:
This doesn't look right to me. We should be applying the basicStyle first and then adjusting it based on finer grained properties. What about having primaryStyle second was causing an issue? I would think that's where we should make a change.

Reply:
`primaryStyle` sets background and color again (through `backgroundStyle` function). `color` is set by `basicStyle` and then primaryStyle overrides it even thought `button.color` is set.